core/src/test/scripts/test_data/data_cna_genecol_presence_both.txt

```diff
@@ -6,3 +6,4 @@ ATAD3B	83858	0	-1	0	-1	2
 ATAD3C	219293	0	-1	0	-1	2
 AURKAIP1	54998	0	-1	0	-1	2
 B3GALT6	126792	0	-1	0	-1	2
+ERCC5	2073	0	-1	0	1.5	2
```

Comment:
you can easily add -1.5 here as well to increase your test coverage from 50% to 100% for this case